tests/component/test.deletion_error.js

```diff
@@ -29,8 +29,8 @@ describe('test.deletion_error.js', function () {
     server = app.listen(0);
   });
 
-  after(function () {
-    return server.close();
+  after(function (done) {
+    server.close(done);
```

Comment:
I think this is safer because the docs say the callback will be called with an Error if the server was not open when it was closed. That would end up failing the `after` which would not be right in this case.

```suggestion
    server.close(() => done());
```

Reply:
I think it's reasonable for the `after()` to fail if the server has already stopped - presumably it means that the server has crashed, which would probably need investigating, especially if it wasn't caught elsewhere.

Reply:
Fair point. This is an improvement either way so approved, and I'll let you pick which you prefer.